app/models/idea.rb

```diff
@@ -1,3 +1,5 @@
 class Idea < ActiveRecord::Base
+  validates :title, format: { with: /\A[a-zA-Z0-9,?! ]+\z/ }
+  validates :body, format: { with: /\A[a-zA-Z0-9,?! ]+\z/ }
```

Comment:
@dastinnette This is a good solution - but I think using JavaScript might be better long term as the fix. You'd want to escape the input on the JS side before you store it.